src/Vasont.Inspire.Models.csproj

```diff
@@ -20,7 +20,7 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>CCMS</PackageTags>
     <PackageReleaseNotes>Added SubmissionAttributeModel</PackageReleaseNotes>
-    <Version>1.3.55</Version>
+    <Version>1.3.56</Version>
```

Comment:
You need to up the version in two other places as well.
   <AssemblyVersion>1.3.55.0</AssemblyVersion>
    <FileVersion>1.3.55.0</FileVersion>